src/test/resources/application.properties

```diff
@@ -1,3 +1,3 @@
 wait.timeout.seconds=15
-logging.pattern.console="%d{HH:mm:ss.SSS} [%thread] [%X{session.id}][%X{request.id}] %-5level %logger{36} - %msg%n%rEx{full, org.springframework.web, org.springframework.boot, org.apache.catalina, org.apache.coyote, org.apache.tomcat, java.base}"
+logging.pattern.console="%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n%rEx{full, org.springframework.web, org.springframework.boot, org.apache.catalina, org.apache.coyote, org.apache.tomcat, java.base}"
```

Comment:
If/when we get this working, it might be a good idea to add the test name to the MDC and add to the logging lines.

Reply:
We have it working in application.yaml in the claimant service:
```
logging:
  pattern:
    console: "%d{HH:mm:ss.SSS} [%thread] [%X{session.id}][%X{request.id}] %-5level %logger{36} - %msg%n%rEx{full, org.springframework.web, org.springframework.boot, org.apache.catalina, org.apache.coyote, org.apache.tomcat, java.base}"
  #        console: "%d{HH:mm:ss.SSS} [%thread] [%X{session.id}][%X{request.id}] %-5level %logger{36} - %msg%n"
  level:
    ROOT: ${vcap.services.variable-service.credentials.claimant-root-loglevel:info}
    uk.gov.dhsc: ${vcap.services.variable-service.credentials.claimant-app-loglevel:debug}
```
I can't see anything wrong with the properties file though

Reply:
Just spotted that this is in test. Have you tried with this in src/main/resources? Not that I can say why that would make a difference.

Reply:
Yes, that is exactly where I copied it from (removing request and session id of course), but it doesn't work. I've just tried adding it to src/main/resources and it has no effect there either. I guess it not the end of the world, just a bit annoying. I'll take another look at it when all the important stuff is done.